flow-report/tsconfig.json

```diff
@@ -19,7 +19,7 @@
     "**/*.ts",
     "**/*.tsx",
     "./types",
-    "../types/test.d.ts",
+    "../types/env/test.d.ts",
```

Comment:
maybe `types/dev/*.d.ts`? Or `types/internal/*.d.ts`? I get that idea of `env`, but it's not the first thing I think of (there's no reason a child project can't inherit an `env`), so the others may be more descriptive

Reply:
`types/internal/*.d.ts` sounds good to me